libraries/mq/client.cpp

```diff
@@ -148,7 +171,7 @@ error_code client_impl::prepare()
       return ec;
    }
 
-   auto queue_res = _reader_broker->declare_queue(
+   auto queue_res = m.declare_queue(
       "",
```

Comment:
Do we want to set the queue name here if it was previously set? Would that enable reconnecting to the results queue.

Reply:
A client RPC receipt queue is `auto-delete` and `exclusive`, so if the client drops connection the queue is gone.